### PR TITLE
Adds Equip call for Chaplain's Bible

### LINKED ISSUE #7076: Chaplain Does not have bible on roundstart
**Problem Description**:
The chaplain does not have his bible before or after completing the prompts on spawn

**What did you expect to happen**: 
He used to spawn with a bible in hand

**What happened instead**:
After completing the prompts, I had no bible to speak of in my inventory or around me

**Why is this bad/What are the consequences:**
It forces the chaplain to have to go to the library and either hop the table to print one or hope the librarian isn't being a dick about it.

**Steps to reproduce the problem**:
Play as a roundstart chaplain

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**
This has happened every time I've played chaplain for some time. The only thing that could potentially prove useful is that I am always an IPC.

### PR DESCRIPTION
**Pull request by IK3I**
Fixes #7076

:cl: IK3I
fix: Chaplain now spawns with his bible
/:cl:

### CODE REVIEW

**Comment by Tayyyyyyy** [comment]
One small thing. Maybe edit the changelog to be "their" instead of "his"?

**Comment by fludd12** [comment]
@Tayyyyyyy As much as I love genderless _They_, given my reputation on the server, _His_ has become conventionally accepted androgynous language in most cases, with _They_ having been abandoned something like a half-century ago and only recently re-adopted. Besides, it's their PR, so they can write the changelog how they please, really.

**Comment by Tayyyyyyy** [comment]
Yeah, just offering the suggestion in case it was an unintentional use of generic he. I understand the changelog of spessman (spessperson?) simulator game isn't crucial to the evolving landscape of gender neutral language. I'll just use generic she in my next PR to balance it out :P

**Comment by Twinmold93** [comment]
Why is gender debate starting on our git? It's literal just ease of quick reference to read and our spessmen game will always do so. Also, thanks for fixing chaplain.

**Comment by Tayyyyyyy** [comment]
There's no debate here. I respect people who like generic he and I understand gaming is a male dominated space. Not trying to step on anyone's toes here; I accepted @fludd12's explanation and made a joking comment about making a balancing PR (I don't really do PRs). "Spessperson" was an attempt at a self deprecating joke, too. Apologies if I've offended anyone.

**Comment by Birdtalon** [comment]
Our github turning into tumblr in this PR.

File: code/game/jobs/job/support_chaplain.dm [hunk 1]
```diff
@@ -34,6 +34,8 @@
 	var/obj/item/weapon/storage/bible/B = new /obj/item/weapon/storage/bible(H)
 
 	spawn()
+		H.equip_to_slot_or_del(B, slot_l_hand)
```
Comment (Fox-McCloud) [inline code/game/jobs/job/support_chaplain.dm]: I'd put this at the veryyyy end, and also put it in the backpack, so it doesn't potentially interfere with other hand equipping things.

`H.equip_to_slot_or_del(B, slot_in_backpack)`

Comment (IK3I) [inline code/game/jobs/job/support_chaplain.dm]: The only problem with that is the Chaplin can change the sprite of the bible when first received. Therefore, it has to be equipped prior to the feedback cycle AND be in a location that can have is appearance updated. If there is an appearance update for the backpack slot, then that's simple. However, equipping prior to customization seems unavoidable.

Comment (Fox-McCloud) [inline code/game/jobs/job/support_chaplain.dm]: Fair enough.

This should probably be a preference in the player menu, eventually.

**Pull request merged**